packages/bits/src/lib/chips/chip/chip.component.ts

```diff
@@ -21,7 +21,10 @@ import { IChipsItem } from "../public-api";
     templateUrl: "./chip.component.html",
     styleUrls: ["./chip.component.less"],
     encapsulation: ViewEncapsulation.None,
-    host: { class: "nui-chip" },
+    host: {
+        class: "nui-chip",
+        role: "option",
```

Comment:
I'm not sure if `option` is correct here. This implies that you can select a chip, but I don't think chips are typically selectable.

Reply:
will change to listitem, thanks